demo/midgard-sdk/src/common.ts

```diff
@@ -243,6 +251,38 @@ export const AddressSchema = Data.Object({
 export type AddressData = Data.Static<typeof AddressSchema>;
 export const AddressData = AddressSchema as unknown as AddressData;
 
+export const fromAddress = (
```

Comment:
Use a more descriptive name, e.g.:
```suggestion
export const addressDataFromBech32 = (
```